src/wp-admin/css/list-tables.css

```diff
@@ -579,12 +579,12 @@ th.sorted.desc:hover .sorting-indicator.asc:before {
 	left: 0;
 }
 
-.check-column .label-covers-full-cell + input {
+.check-column input {
 	position: relative;
 	z-index: 1;
 }
 
-.check-column .label-covers-full-cell:hover + input {
+.check-column:hover input {
 	box-shadow: 0 0 0 1px #2271b1;
```

Comment:
Removing this style entirely might be better than using a new style.

Reply:
I just noticed that this PR puts the spinner between the checkbox and the label in Privacy requests tables, so the next sibling selector would not work (if considering any style like what I mentioned above, it would need `input:hover ~ label` instead).

Reply:
Makes sense to me to move the spinner. As far as Contact Form 7 is concerned, I don't think we should worry too much about how an already inaccessible plugin is supported here. The fix there is for CF7. We should try to avoid any egregious problems for plugins, but we can't solve their issues for them.

Reply:
Keeping the hover state on the input is fairly trivial; updated CSS to do that. The background is somewhat useful; it offers affordance for sighted users in addition to the checkbox state. I'm not sure it's necessary - added it, though softened somewhat, since it's not the sole indicator.  Not sure it's required.

Reply:
If the label is either non-existent or visually hidden with the `screen-reader-text` class, then clicks outside a checkbox would not change its state. When hovering over cells like those in custom list tables, the checkbox would have the `box-shadow` without the background change. This distinction is not as much as I would like, yet it might be enough to avoid confusion.

When using the background color, padding can make the label too big for the cell. Setting padding to zero could require a higher specificity, but `box-sizing: border-box` should prevent going beyond the 100% in either direction.

I found another plugin's custom list table: TablePress updated the checkbox label to use the `label-covers-full-cell` class. That behaves the same way as the Core table checkbox cells with this PR.

Reply:
Hi @sabernhardt,

Tobias, developer of TablePress, here. I added that `class` after https://core.trac.wordpress.org/changeset/55954, from what I can see in my commit history. I'll be happy to make necessary adjustments here, of course!

Reply:
Tobias, I'm glad to have the example, and your list table is consistent with Core tables now. Once these changes are available in a public version, you may want to move the checkbox above the label and/or remove the class name (but probably not before then).

Reply:
@sabernhardt, thanks for confirming! Yes, I'll be moving the checkbox then.